src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const trs = document.querySelectorAll('tr');
+
+trs.forEach((el) => {
+  const parentElement = el;
+  const firstChild = parentElement.children[1];
+  const clonedChild = firstChild.cloneNode(true);
+
+  parentElement.insertBefore(clonedChild, parentElement.children[4]);
```

Comment:
The task requires appending the cloned column in front of the last table column. The current code inserts the cloned node before the fifth child of each row. Make sure to adjust this logic to correctly append the cloned column in front of the last column.